lib/chef/knife/ec_backup.rb

```diff
@@ -91,11 +98,13 @@ def for_each_user
 
       def for_each_organization
         rest.get('/organizations').each_pair do |name, url|
+          $organizations_hash["total"]=$organizations_hash["total"]+1
```

Comment:
```suggestion
          next unless (config[:org].nil? || config[:org] == name)
          $organizations_hash["total"]=$organizations_hash["total"]+1
```
I'm not sure, but do we want this instead?

Reply:
(Maybe we should ask Marc)